app/logical/bulk_update_request_processor.rb

```diff
@@ -182,6 +198,16 @@ def process!
           tag = Tag.find_or_create_by_name(args[0])
           tag.update!(category: Tag.categories.value_for(args[1]))
 
+        when :deprecate
+          tag = Tag.find_or_create_by_name(args[0])
+          tag.update!(is_deprecated: true)
+          TagImplication.active.where(consequent_name: tag.name).each { |ti| ti.reject!(User.system) }
+          TagImplication.active.where(antecedent_name: tag.name).each { |ti| ti.reject!(User.system) }
+
+        when :undeprecate
+          tag = Tag.find_or_create_by_name(args[0])
+          tag.update!(is_deprecated: false)
+
```

Comment:
Should the tag implications be reactivated? I'm thinking of the case where a mistake was made, or something else where the original status needs to be restored.

Reply:
I don't think mistakes in BUR are likely to happen, so if they did it they can just be readded manually.